src/main/java/com/Novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,56 @@
+package com.Novi.TechItEasy.controllers;
+
+import com.Novi.TechItEasy.exceptions.NameTooLongException;
+import com.Novi.TechItEasy.exceptions.RecordNotFoundException;
+import jakarta.websocket.server.PathParam;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
+public class TelevisionController {
+
+    String[] televisionBrands = {"Sony", "Phillips", "LG"};
+    List<String> televisionDatabase = new ArrayList<>(List.of(televisionBrands));
```

Comment:
Het is prima dat je je "database" al vast vult, maar er zijn beter manieren om dat te doen. Een TelevisionController constructor zou in dit geval de beste keus zijn.